js/menu.js

```diff
@@ -0,0 +1,10 @@
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+
+openNav();
+closeNav();
```

Comment:
Please kindly desist from deleting review comments by previous reviewers, as it leads to inconsistency and confusion in reviews given on your PR and further attempts will lead to strict measures being taken on you by senior reviewers. kindly note that all reviews given are to help you have good code quality, will help you in your job search and will enable you to be a better programmer so keep pushing 😉

Reply:
Alright, but sometimes it can be duplicates or things that are not clear. Anway, I will consult the reviewer in case I case a doubt of the review comments.